cmatrix.c

```diff
@@ -181,6 +182,14 @@ void var_init() {
         matrix[i] = matrix[i - 1] + COLS;
     }
 
+    for (i = 0; i < LINES; ++i)
+    {
+        for (j = 0; j < COLS; ++j)
+        {
```

Comment:
I am skeptical about the need of this loop

Reply:
You are right... Fixed.